yarn.lock

```diff
@@ -6210,7 +6210,6 @@ npm@^8.3.0:
     "@npmcli/fs" "^2.1.0"
     "@npmcli/map-workspaces" "^2.0.3"
     "@npmcli/package-json" "^2.0.0"
```

Comment:
Can we undo these changes? They're not related to the TS update

Reply:
Updating TypeScript actually makes these changes to `yarn.lock`. Do you still want me to undo?

Reply:
Interesting. Can you manually add the three lines back? 6213, 6224 and 6244

Reply:
Done!